### DISCUSSION TITLE
API End point

### DISCUSSION THREAD

**Discussion by devsoftzz** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#25] [Ideas]
Official Doc: https://www.django-rest-framework.org/
Watch 2nd,3rd,4th Video of this Playlist: https://www.youtube.com/playlist?list=PLgCYzUzKIBE9Pi8wtx8g55fExDAPXBsbV

**Comment by shivam1shroff** [comment]
I've found another way to get JSON response as an API result.
Here is the link for that :- https://www.youtube.com/watch?v=Jzv3G5iDLvw
Although it uses Flask still does our job.

**Reply by devsoftzz** [reply]
As both of the methods allows JSON responses, Use any of the methods which you think are more suitable for our project.
Don't go with XML or other response formates. As JSON is more versatile and easy to work with it.

**Reply by shivam1shroff** [reply]
Yes @devsoftzz, I know that how easy it is to work with JSON. And I guess I'll go with your suggested approach because our other codes are written in Django and it would be much more easier to continue with that rather than introducing new tool and then making connections in between them.

**Comment by shivam1shroff** [comment]
I've found this GitHub Repo in which we can find all the information we want for RestAPI.
It is quite impressive repo almost all the information about Rest framework is given in very easy to understanding way.
https://github.com/pymike00/awesome-django-rest-framework

**Comment by shivam1shroff** [comment]
Here I've found another link that provides us How to make API Endpoints including how to test them.
https://www.caktusgroup.com/blog/2019/02/01/creating-api-endpoint-django-rest-framework/
